src/components/button/button.ts

```diff
@@ -37,7 +37,7 @@ export class MdButton {
   /** Whether a mousedown has occurred on this element in the last 100ms. */
   isMouseDown: boolean = false;
 
-  setClassList() {return `md-${this.color}`;}
+  setClassList() {return `md-${this.color}`; }
```

Comment:
This can not currently be enforced, there is an enhancement tslint issue open that would eventually cover this. I fixed the style issue